packages/reaction-collections/server/publications/products.js

```diff
@@ -15,24 +15,30 @@ Meteor.publish("Products", function (productScrollLimit, shops) {
   }
   let Products = ReactionCore.Collections.Products;
   let limit = productScrollLimit || 10;
+  let sort = {title: 1};
+
   if (shop) {
-    let selector = {
-      shopId: shop._id
-    };
-    // handle multiple shops
-    if (shops) {
-      selector = {
-        shopId: {
-          $in: shops
-        }
-      };
-      // check if this user is a shopAdmin
-      for (let thisShopId of shops) {
-        if (Roles.userIsInRole(this.userId, ["admin", "createProduct"],
-            thisShopId)) {
-          shopAdmin = true;
+    let selector = {shopId: shop._id};
+
+    if (productFilters) {
+      // handle multiple shops
+      if (productFilters.shops) {
+        check(productFilters.shops, Array);
```

Comment:
I think we should check format of this argument as usual we do in the top of all publications.
Yes, I checked, AND is not necessary here